Pcap++/header/PcapFilter.h

```diff
@@ -278,9 +278,8 @@ namespace pcpp
 
 	/**
 	 * @class IPFilter
-	 * A class for representing IPv4 address filter, equivalent to "net src x.x.x.x" or "net dst x.x.x.x"<BR>
+	 * A class for representing IPv4 or IPv6 address filter, equivalent to "net src x.x.x.x" or "net dst x.x.x.x"<BR>
 	 * For deeper understanding of the filter concept please refer to PcapFilter.h
-	 * @todo Add IPv6 filtering support
 	 */
```

Comment:
Removed todo and added documentation about IPv6 support as it is supported since #1324